src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -3878,6 +3325,23 @@ namespace VariableSpeedCoils {
         }
     }
 
+    void checkVarSpeedCoilCoolCap(EnergyPlusData &state, std::string coilName, Real64 const highestSpeedCoolCap, Real64 &totalCoolCap)
```

Comment:
@RKStrand Is this the only substantive change? A walkthrough highlighting the real changes would be helpful.

Reply:
Yes, that is the only real change--a call to this new routine to sort out the issue as I saw it.